docs/end_to_end_tutorials/dev_practices/e2e_evaluation.md

```diff
@@ -0,0 +1,41 @@
+# End-to-End Evaluation
+## Setting up an Evaluation Set
+
+To help with this, we've created some tools that automatically generate a dataset for you given a set of documents to query.
```

Comment:
As a page on its own, starting with "To help with this" feels a bit awkward since it's the first sentence on this page 

Could maybe use an intro paragraph?

Reply:
The right procedure IMO is not actually to start with component-wise testing, rather to start with e2e testing to get a sense of how the entire application is doing. 

Only when it is verified that the application as a whole works/doesn't work well does it make sense to do more rigorous testing of individual components